.env

```diff
@@ -20,7 +20,8 @@ CANTALOUPE_TAG=v1.0.0
 MANIFEST_TAG=v1.0.0
 
 # Solr
-SOLR_URL=http://solr:8983/solr
+SOLR_HOST=http://solr:8983/solr
```

Comment:
these changes (all of the .env / .circle / Gemfile ones) do not appear to be related to this PR.  Why are they here and not in their own PR?

Reply:
I'm moving them to their own PR. We were working on it in service of getting this PR passing CI, but I think they belong on their own branch. Working on it now.